packages/cmd/src/response/CommandResponse.ts

```diff
@@ -800,8 +804,8 @@ export class CommandResponse implements ICommandResponseApi {
                         outer.writeStream(daemonHeaders);
 
                         outer.mProgressBar.terminate();
-                        outer.writeStdout(outer.mStdout);
-                        outer.writeStderr(outer.mStderr);
+                        outer.writeStdout(outer.mStdout.subarray(this.mProgressBarStdoutStartIndex));
+                        outer.writeStderr(outer.mStderr.subarray(this.mProgressBarStderrStartIndex));
```

Comment:
Important note:
- In `next` we were not writing the subarray
- In `master`, we were only writing to `process.std` and not to `mStream`

Reply:
Added @dkelosky and @awharn  since they were involved with the progress bar in either `next` or `master` 😋

Reply:
thank you for the fyi - the code change makes sense after talking with @zFernand0